### PR TITLE
Students 

### PR DESCRIPTION
**Pull request by Annishivakumara**
**Updated YouTube reference section**
> Added a link to the tutorial video and encouraged students to subscribe for more content.

**Added images** 
>  Included images to enhance the visual appeal of the tutorial and provide better understanding.

### CODE REVIEW

File: notes/1.md [hunk 1]
```diff
@@ -78,3 +78,15 @@ print("Namaste, nanna hesaru [Your Name]!")
...
 3. **Practice** running your Python code through both an IDE and a terminal.
 4. **Build in Public** Create a post on Linkedin/X and share that you are starting the course and its day 1. (Use #engineeringinkannada and mention me)
+
+### **YouTube Reference**
+watch the following YouTube video from my channel:
+- [Watch the tutorial on YouTube](https://youtu.be/8c74mXV2lJ0?si=KHrJ3pbPaYkkInQk)
+
+
+
+![Python Programming Tutorial](image-1.png)
```
Comment (chandansgowda) [inline notes/1.md:88]: we can remove this image

File: notes/1.md [hunk 2]
```diff
@@ -78,3 +78,15 @@ print("Namaste, nanna hesaru [Your Name]!")
 ```
 3. **Practice** running your Python code through both an IDE and a terminal.
 4. **Build in Public** Create a post on Linkedin/X and share that you are starting the course and its day 1. (Use #engineeringinkannada and mention me)
+
+### **YouTube Reference**
+watch the following YouTube video from my channel:
```
Comment (chandansgowda) [inline notes/1.md:83]: W in Caps

**Review by chandansgowda — CHANGES_REQUESTED** [review]
We can just have Youtube Video Link - <link> 
And this has to be done for all notes.

File: notes/1.md (hunk 1 above)
Comment (Annishivakumara) [reply notes/1.md:88]: ok 
i will make changes ..

**Pull request closed without merging**